packages/address-book-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Changed
+- **BREAKING:** Add `addressType` flag to address book entries ([#828](https://github.com/MetaMask/controllers/pull/828))
+  - Each existing address entry will need to be updated to set the `addressType` field to `AddressType.unknown`.
+- Rename this repository to `core` ([#1031](https://github.com/MetaMask/controllers/pull/1031))
+- Update `@metamask/controller-utils` package ([#1041](https://github.com/MetaMask/controllers/pull/1041)) 
```

Comment:
Should this (and similar entries) be marked as breaking? I'm wondering if this seems too innocuous. Should we advise the reader to review the release notes for this version, or do you think that will be obvious considering that the changes for the new version of `controller-utils` will be included later on in the release notes?

Reply:
Good point. The update alone doesn't seem like a breaking change, but the indirect effects probably are. I will review this to see which packages rely on the functions with breaking changes.

Reply:
I have reviewed uses of these constants, and the only modules affected are the currency rate controller and the network controller. The effects to both of these modules is already described adequately in their respective package changelogs. As for the breaking changes to `getBuyUrl`, that isn't used directly in this repo, so we're good there as well.